src/Illuminate/Cache/MemcachedStore.php

```diff
@@ -49,6 +49,35 @@ public function get($key)
     }
 
     /**
+     * Retrieve multiple items from the cache by key.
+     *
+     * Items not found in the cache will have a null value for the key.
+     *
+     * @param  array  $keys
+     * @return array
+     */
+    public function getMultiple(array $keys)
+    {
+        $prefixedKeys = [];
+
+        foreach ($keys as $keyToPrefix) {
+            $prefixedKeys[] = $this->prefix.$keyToPrefix;
+        }
+
+        $cas = null;
+        $flags = 1; //\Memcached::GET_PRESERVE_ORDER
```

Comment:
I did initially, it failed the test due to the module not being installed.  How do you suggest handling this?